Userland/Applications/Calculator/main.cpp

```diff
@@ -64,6 +64,19 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
         widget->set_entry(KeypadValue { Crypto::SignedBigInteger::create_from(27182818284), 10 });
     }));
 
+    auto& round_menu = window->add_menu("&Round");
+    round_menu.add_action(GUI::Action::create("&0", [&](auto& action) {
```

Comment:
What does this even mean?
Round to 0 digits?
Round towards 0?
Round away from 0?

(please make it say "To zero digits" or something, just "zero" is fairly ambiguous)

Reply:
Do you want me to do it for all three actions ?

Reply:
sure, yeah

Reply:
Corrected, is it good for you ?